vendor/vendor-android/src/main/kotlin/com/malinskiy/marathon/android/executor/AndroidDeviceTestRunner.kt

```diff
@@ -29,7 +30,7 @@ class AndroidDeviceTestRunner(private val device: AndroidDevice) {
     companion object {
         private val JUNIT_IGNORE_META_PROPERTY = MetaProperty("org.junit.Ignore")
         private const val MAX_TEST_DURATION_LEEWAY = 1.2 // +20% to expected test duration
-        private const val BATCH_DURATION_LEEWAY = 1.1 // +10% to expected batch duration
+        private const val BATCH_DURATION_LEEWAY = 1.3 // +30% to expected batch duration
```

Comment:
These _almost_ look like something that should be defined in configuration file.

Reply:
And it is. Actually, my solution is that timeout is based on the biggest value - either from historical data (combined for batch) or from configuration.

Reply:
I just increased the batch timeout in Marathon config (different repo).